### PR TITLE
Put vip-mysql2.osuosl.bak on eth1 (backend) interface

### PR DESCRIPTION
**Pull request by TravisWhitehead**
This moves `vip-mysql2.osuosl.bak` to the correct (backend) interface, from `eth0` to `eth1`.

ServerSpec tests are currently failing due to `eth1` not existing in Kitchen VMs.
@ramereth Do you know of a good way of creating another network interface for the sake of testing this? I would guess it would depend on OpenStack settings, but I'm unsure.

### CODE REVIEW

**Comment by TravisWhitehead** [comment]
[keepalived](https://trello.com/c/68d7MKNW/236-keepalived)

**Comment by ramereth** [comment]
@TravisWhitehead [here](https://github.com/osuosl-cookbooks/osl-dns/blob/master/test/cookbooks/dns-test/recipes/default.rb) is a good example of how to do it. Basically you use the dummy module to create a fake interface and then bring that interface up.

**Comment by TravisWhitehead** [comment]
It looks like the suite is failing to switch to MASTER in serverspec, but works in chef-provisioning. Will have to investigate.

**Comment by ramereth** [comment]
@TravisWhitehead it's because you're using the FQDN in the logic for the recipes. I tried the other approach with using the ``server_name`` via cloud-init, but that doesn't work since we have a fake domain name setup with OpenStack. We can likely work around that sometime, but I'm worried it will cause other problems.

The other approach that I was able to get working was using the [chef_hostname](https://github.com/chef-boneyard/chef_hostname) cookbook (which is deprecated when we switch to Chef 14, but ok for Chef 12/13). Here's the following changes I made to get this working. Keep in mind you'll need to update the tests.

``` patch
diff --git a/.kitchen.yml b/.kitchen.yml
index da698fc..02c7bec 100644
--- a/.kitchen.yml
+++ b/.kitchen.yml
@@ -10,7 +10,11 @@ suites:
       - recipe[osl-keepalived::default]
   - name: haproxy_osuosl
     run_list:
+      - recipe[keepalived_test::hostname]
       - recipe[osl-keepalived::haproxy_osuosl]
+    attributes:
+      keepalived_test:
+        fqdn: lb1.osuosl.org
   - name: haproxy_phpbb
     run_list:
       - recipe[osl-keepalived::haproxy_phpbb]
diff --git a/test/cookbooks/keepalived_test/metadata.rb b/test/cookbooks/keepalived_test/metadata.rb
index 125d45d..b61b59d 100644
--- a/test/cookbooks/keepalived_test/metadata.rb
+++ b/test/cookbooks/keepalived_test/metadata.rb
@@ -7,3 +7,4 @@ long_description ''
 version          '0.1.0'
 depends          'base'
 depends          'osl-keepalived'
+depends          'chef_hostname'
diff --git a/test/cookbooks/keepalived_test/recipes/hostname.rb b/test/cookbooks/keepalived_test/recipes/hostname.rb
new file mode 100644
index 0000000..3c099df
--- /dev/null
+++ b/test/cookbooks/keepalived_test/recipes/hostname.rb
@@ -0,0 +1 @@
+hostname node['keepalived_test']['fqdn']
```

**Comment by TravisWhitehead** [comment]
@ramereth @jcorme Ready for re-review

File: metadata.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@
 long_description 'Installs/Configures osl-keepalived'
 version          '0.2.0'
 
+depends          'chef_hostname'
```
Comment (ramereth) [inline metadata.rb]: This should be in the test cookbook metadata.

**Comment by TravisWhitehead** [comment]
@ramereth Moved the dependency to the test cookook, and added some notes in the README.

**Comment by TravisWhitehead** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v0.2.1.  Have a nice day!

**Pull request merged**